MainRepository/Program.cs

```diff
@@ -1,7 +1,20 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿while (true) {

```

Comment:
There is now way to exit the program. Maybe if the user types 'q'/'Q', you can exit it.